env_vars.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Environment variables for build
+MACOSX_DEPLOYMENT_TARGET=10.9
+# CFLAGS override for https://github.com/pandas-dev/pandas/issues/34114
+if [ "$(uname)" == "Linux" ]; then
+    CFLAGS="-ansi -Wno-error=maybe-uninitialized -Wno-error=sign-compare -Wno-error=return-type"
+fi
+# Macos's linker doesn't support stripping symbols
+if [ "$(uname)" != "Darwin" ]; then
```

Comment:
Is this just an attempt to reduce the binary size? Feature instead of fixing an issue?